.github/ISSUE_TEMPLATE/release.md

```diff
@@ -19,20 +19,20 @@ Release builds URL: ???
 * [ ] Confirm the application opens (dismiss the updater)
 * [ ] Create a project with a basic Text-to-CAD prompt
 * [ ] Confirm the result is viewable in an engine stream
-* [ ] Open the application again and confirm the updater can downgrade
+* [ ] Use 'Check for updates' to bring back the updater toast and confirm the app can update to the previous release
```

Comment:
So @jacebrowning this is what I had in mind:
1. you open the release candidate (eg. 1.0.6)
2. you dismiss the updater toast to the previous release (eg. 1.0.5)
3. you do the other checks
4. you hit 'Check for updates' using one of the buttons
5. you hit accept the update in the toast and confirm the app 'updates' to the previous release (eg. 1.0.5)

Reply:
Yeah, that makes sense! I might split this up into two steps but up to you:

5. Check for app updates and click restart
6. Confirm the previous version of the app is installed